task/bentoml/anchore-cli/0.1/README.md

```diff
@@ -0,0 +1,80 @@
+# Anchore CLI
+
+The Anchore CLI provides a command line interface on top of the Anchore Engine REST API.
+Anchore CLI will try to connect to the Anchore Engine at http://localhost/v1 with no authentication. The username, password and URL for the server can be passed to the Anchore CLI as command line arguments
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/anchore-cli/0.1/anchore-cli.yaml
+```
+
```

Comment:
any reason for placing the `anchore-cli` task in `bentoml` task ?

Reply:
no.
May be file was added accidentally

Reply:
@arunvel1988 in that case, can you please remove it?

Reply:
@vinamra28  removed